src/deep-fs/lib/FS.js

```diff
@@ -120,14 +134,28 @@ export class FS extends Kernel.ContainerAware {
    * Returns mounted file system folder (tmp, public or system)
    *
    * @param {String} name
-   * @returns {S3FS|SimulatedS3FS|*}
+   * @param {String} msIdentifier
+   * @returns {fs|s3fs|S3FS|S3FsRumProxy|SimulatedS3FS|*}
    */
-  getFolder(name) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [This function's cyclomatic complexity is too high. (11)](https://www.codacy.com/app/MitocGroup/deep-framework/file/2210381231/issues/source?bid=2895475&fileBranchId=3171000#l140)